pkgs/servers/monitoring/nagios/plugins/smartmon.nix

```
@@ -0,0 +1,39 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, python2
+, smartmontools
+}:
+
+stdenv.mkDerivation {
+  pname = "check_smartmon";
+  version = "master";
+
+  src = fetchFromGitHub {
+    owner  = "check-plugins";
+    repo   = "check_smartmon";
+    rev    = "07b14b8eae3c0f2f6193eca5c208708422cfff32";
+    sha256 = "18120zi0xr1vwza80n8gcjciklk7jgg7w4cnghnnbxrcirv33fih";
+  };
+
+  buildInputs = [ (python2.withPackages (pp: [ pp.psutil ]))
+                  smartmontools
+                ];
```

```suggestion
  buildInputs = [ 
    (python2.withPackages (pp: [ pp.psutil ]))
    smartmontools
  ];
```

Remembering that Python 2 is unsupported. If you can provide some Python 3 replacement for this, it will be way more easily accepted.

---

(also, looking at the project, it looks a bit abandoned...)

---

Fortunately, check_smartmon has been maintained. I updated the package with a new repository and updated to Python 3.